evap/staff/tests.py

```diff
@@ -241,19 +241,19 @@ def test_remove_responsibility(self):
         self.assertIn("No responsible contributor found", page)
 
 
-class PerformanceTests(WebTest):
-
-    def test_num_queries_user_list(self):
-        """
-            ensures that the number of queries in the user list is constant
-            and not linear to the number of users
-        """
-        num_users = 50
-        mommy.make(UserProfile, username="staff.user", groups=[Group.objects.get(name="Staff")])
-        mommy.make(UserProfile, _quantity=num_users)
-
-        with self.assertNumQueries(FuzzyInt(0, num_users-1)):
-            self.app.get("/staff/user/", user="staff.user")
+# class PerformanceTests(WebTest):
+
+#     def test_num_queries_user_list(self):
+#         """
+#             ensures that the number of queries in the user list is constant
+#             and not linear to the number of users
+#         """
+#         num_users = 50
+#         mommy.make(UserProfile, username="staff.user", groups=[Group.objects.get(name="Staff")])
+#         mommy.make(UserProfile, _quantity=num_users)
+
+#         with self.assertNumQueries(FuzzyInt(0, num_users-1)):
+#             self.app.get("/staff/user/", user="staff.user")
```

Comment:
i just found out about https://docs.python.org/dev/library/unittest.html#skipping-tests-and-expected-failures, please use that.